### PR TITLE
Make reuse conformant

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Everything except changelog/fragments/ is now reuse conformant (https://reuse.software/spec/).

CC @gotmax23

##### ISSUE TYPE
- Docs Pull Request
- Feature Pull Request

##### COMPONENT NAME
licensing

### CODE REVIEW

**Comment by gotmax23** [comment]
> Everything except changelog/fragments/

If you wanted, you could add a `.reuse/dep5` file to the repo root with the following contents:

```
Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/

Files: changelogs/fragments/*
Copyright: Ansible Project
License: GPL-3.0-or-later
```

Then, the entire repository would be fully REUSE compliant[^1] and [it could get a REUSE compliant badge](https://reuse.software/dev/#api).

[^1]: without having to add headers that take up more room than the actual changelog fragment

**Comment by felixfontein** [comment]
@gotmax23 great idea, I've added that file.

File: README.md [hunk 1]
```diff
@@ -133,6 +133,10 @@ See [this issue](https://github.com/ansible-collections/community.general/issues
...
 
-GNU General Public License v3.0 or later.
+This collection is primarily licensed and distributed as a whole under the GNU General Public License v3.0 or later.
 
-See [COPYING](https://www.gnu.org/licenses/gpl-3.0.txt) to see the full text.
+See [LICENSES/GPL-3.0-or-later.txt](https://github.com/ansible-collections/community.general/blob/main/COPYING) for the full text.
+
+Parts of the collection are licensed under the [BSD 2-Clause license](https://github.com/ansible-collections/community.general/blob/main/LICENSES/BSD-2-Clause.txt), the [MIT license](https://github.com/ansible-collections/community.general/blob/main/LICENSES/MIT.txt), and the [PSF 2.0 license](https://github.com/ansible-collections/community.general/blob/main/LICENSES/PSF-2.0.txt).
+
+All files except changelog fragments (that will not be part of a release) have a machine readable `SDPX-License-Identifier:` comment denoting its respective license(s), or an equivalent entry in an accompanying ``.license`` file. This conforms to the `REUSE specification <https://reuse.software/spec/>`_.
```
Comment (gotmax23) [inline README.md:142]: You may want to tweak this a bit now that there's the `dep5` file for the changelog fragments, but this is approved.

**Review by gotmax23 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (felixfontein) [reply README.md:142]: How about:
```suggestion
All files have a machine readable `SDPX-License-Identifier:` comment denoting its respective license(s), or an equivalent entry in an accompanying `.license` file. Only changelog fragments (which will not be part of a release) are covered by a blanket statement in `.reuse/dep5` This conforms to the [REUSE specification](https://reuse.software/spec/).
```
?

(I also noticed I once again started writing RST in a MarkDown file... meh)

Comment (gotmax23) [reply README.md:142]: > All files have a machine readable `SDPX-License-Identifier:` comment denoting its respective license(s), or an

You don't need a comma before `or`. I didn't notice that before. Besides that, LGTM :).

**Review by gotmax23 — APPROVED** [review]

**Comment by felixfontein** [comment]
@gotmax23 thanks for reviewing this!

**Comment by mxmehl** [comment]
Great job, thanks for making this effort!

What do you think about getting a badge for this? The REUSE projects offers registering a repo with their API: https://api.reuse.software

**Pull request merged**